docs/getting-started/what-is-dotnet.md

```diff
@@ -1,6 +1,150 @@
+
+- Universal Windows Platform (UWP) applications. The Universal Windows Platform allows you to build a single application that can run on the Windows desktop, Windows tablet devices, and Windows Phone. These applications can also be placed in the Windows Store. UWP applications are compiled to native code for their target platforms by .NET Native.   
+
+# .NET Core Components #
+
+Very much like the .NET Framework on the Windows desktop, .NET Core consists of a common language runtime, which in .NET Core is named CoreCLR. .NET Core also features an extensive class library. Rather than a single .NET Framework Class Library, however, .NET Core features CoreFX, a modular collection of libraries. This allows you to include the libraries that your application needs without the overhead of including those that you don't need.
+   
+## The Common Language Runtime ##
+
+CoreCLR, or the common language runtime found in .NET Core, is a lightweight runtime that offers many of the same services as the .NET Framework's common language runtime on the Windows desktop. These include:
+
+- A garbage collector, which provides automatic memory management. The garbage collector allocates and frees memory as needed; you do not have do not it programmatically. .NET Core uses the same garbage collector as the .NET Framework. For more information, see [Garbage Collection](https://msdn.microsoft.com/en-us/library/0xy59wtx(v=vs.110).aspx "Garbage Collection"). 
+
+- A just-in-time (JIT) compiler. which compiles IL, or .NET intermediate language, to native machine code at runtime. On some architectures, the JIT compiler includes support for SIMD hardware acceleration.
+
+- An exception handling mechanism. The runtime, as well as library code, throws exceptions, which you can handle with try/catch statements.   
+
+##The Class Library. 
+
+The class library available in .NET Core is very much like the .NET Framework Class Library for the Windows desktop, with one major difference. Whereas on the Windows desktop is a monolithic library that is part of the operating system and is maintained by Windows Update, on .NET Core it is a modular and factored collection of individual libraries organized by functionality. 
+
+One library, the Microsoft.NetCore.App, is included with the runtime and contains the basic types needed for all application development. Some of these include:
+
+-  Primitives, such as the Boolean type, signed and unsigned integral types, floating-point types, the Char structure.
+  
+-  General-purpose exception classes, such as the ArgumentException, ArgumentNullException, and NullReferenceException types.
+
+- The task types, such as Task and Task<T>, to support asynchronous programming.
```

Comment:
Markdown eats the `<>`, so this is rendered as:

> The task types, such as Task and Task, to support asynchronous programming.

Either write `Task&lt;T&gt;`, or enclose all type names in <code>`</code>.